dashboard/src/main/resources/css/dashboard.css

```diff
@@ -43,20 +43,84 @@
  }
  
  h1 {
-   font-size: 36pt;
+   font-size: 3em;
  }
  
  h2 {
-   font-size: 30pt;
+   font-size: 2.5em;
  }
  
  h3 {
-   font-size: 24pt;
+   font-size: 2em;
  }
+ 
  th, td {
    padding: 5pt;
  }
  
  pre {
    line-height: normal;
- }
\ No newline at end of file
+ }
+ 
+
+ /* ----- Statistic ----- */
```

Comment:
Would you move statistic related classes before `body`?

The file is organized as 

- classes
- html tags